duckduckgo/iOS
# retrieve icons directly from sites

Task/Issue URL: https://app.asana.com/0/488551667048375/1182893760422787
Tech Design URL:
CC:
Fixes: #666 

**Description**:

Remove DDG's favicon service and only images downloaded from the actual host for bookmarks, favorites, tabs, fireproofed sites, etc.

Consolidate favicon loading in to a single place.

**Steps to test this PR**:

1. Smoke test the app, in particular check icons appear as expected in the following places:
* Bookmarks (when viewing bookmarks screen)
* Favorites (when viewing them on the home screen)
* Tabs switcher (when viewing tab switcher)
* Fireproof websites (when viewing settings for Fireproofed websites)

2. Using a proxy of some kind ensure no callback to the favicon service url



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
